google-cloud-translate-v2/CHANGELOG.md

```diff
@@ -0,0 +1,166 @@
+# Release History
+
+## Previous releases of the google-cloud-translate gem
```

Comment:
Hmmm, I think I would prefer to leave out this history, and just start with a blank slate. But that's just opinion.

Reply:
Ehhhh... I don't feel strongly. I'll remove it.